sql-parser/dialect/oracle/src/main/java/org/apache/shardingsphere/sql/parser/oracle/visitor/statement/impl/OracleStatementSQLVisitor.java

```diff
@@ -519,7 +519,7 @@ private ASTNode createAggregationSegment(final AggregationFunctionContext ctx, f
     }
     
     private Collection<ExpressionSegment> getExpressions(final AggregationFunctionContext ctx) {
-        return null == ctx.expr() ? Collections.emptyList() : Collections.singletonList((ExpressionSegment) visit(ctx.expr()));
+        return null == ctx.exprs() ? Collections.emptyList() : Collections.singletonList((ExpressionSegment) visit(ctx.exprs()));
```

Comment:
Hi @Qianyi951015 , I found that there is not method to visit ExprsContext, so this line will not give the expected result.

Reply:
I have corrected it and pushed again

Reply:
OK, thank you！